pkg/networking/networking.go

```diff
@@ -87,7 +95,20 @@ func (n *networkImpl) AddHeaderField(key, value string) {
 
 func (n *networkImpl) AddHeaders(request *http.Request) error {
 	n.addDefaultHeader(request)
-	return middleware.AddAuthenticationHeader(n.GetAuthenticator(), n.config, request)
+	hc := &middleware.HeaderCapture{}
+	var rt http.RoundTripper = hc
+	rt = n.addMiddlewaresToRoundTripper(rt)
+	_, err := rt.RoundTrip(request)
+	for k, v := range hc.CapturedHeaders {
+		request.Header.Set(k, v)
+	}
+
+	return err
+}
+
+func (n *networkImpl) SetUserAgent(userAgent middleware.UserAgentInfo) error {
+	n.snykAppEnvironment = &userAgent
```

Comment:
why is this called snykAppEnvironment, if it's a `UserAgentInfo`?

Reply:
Changed to naming to be consistent 👍